.travis.yml

```diff
@@ -1,4 +1,5 @@
-osx_image: xcode7.3
+osx_image: xcode8
 language: objective-c
 cache: cocoapods
-script: xcodebuild -workspace Demo.xcworkspace -scheme Tests -sdk iphonesimulator build test GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES clean test | xcpretty -c && exit ${PIPESTATUS[0]}
+before_script: pod update
```

Comment:
You need `pod update` since `pod install` doesn't update the specs repo if needed.